playbooks/roles/edxapp/files/update_theme.sh

```diff
@@ -3,28 +3,31 @@
 # Licensed under the MIT license. See LICENSE file on the project webpage for details.
 
 set -x
-
-# Remove if themes folder exists
-if [[ -d /edx/app/edxapp/themes ]]; then
-  sudo rm -fr /edx/app/edxapp/themes
+# If comprehensive theming is enabled then install it, otherwise do nothing
+# EDXAPP_ENABLE_COMPREHENSIVE_THEMING: true
+if [ "$1" == "true" ] || [ "$1" == "True" ]; then
+	# Remove if themes folder exists
+	if [[ -d /edx/app/edxapp/themes ]]; then
+	  sudo rm -fr /edx/app/edxapp/themes
+	fi
+
+	# Create themes folder
+	sudo mkdir /edx/app/edxapp/themes
+	cd /edx/app/edxapp/themes
+
+	# Download comprehensive theming from github to folder /edx/app/edxapp/themes/comprehensive 
+	sudo git clone https://github.com/microsoft/edx-theme.git comprehensive
+	cd comprehensive
```

Comment:
We are assuming the theme name to be used. This should be setup as a parameter as well.